benchmark/opperf/nd_operations/binary_operators.py

```diff
@@ -35,7 +35,34 @@
 
 from benchmark.opperf.utils.benchmark_utils import run_op_benchmarks
 from benchmark.opperf.utils.op_registry_utils import get_all_broadcast_binary_operators, \
-    get_all_elemen_wise_binary_operators
+    get_all_elemen_wise_binary_operators, get_all_misc_binary_operators
+
+
+def run_mx_binary_misc_operators_benchmarks(ctx=mx.cpu(), dtype='float32', profiler='native', warmup=25, runs=100):
```

Comment:
Just curious what falls into the *misc* operator category? Could you provide more info

Reply:
For now, just the two additional ops I implemented (`choose_element_0index` and `reshape_like`), since they did not appear to fall into the broadcast or elementwise binary ops categories.